File: songAPI/Models/Album.cs
using Microsoft.EntityFrameworkCore;

namespace songAPI.Models
{
    public class Album
    {
        public int Id { get; set; }
        public string AlbumName { get; set; } = "";
        public List<Title> TrackList {get; set; } = new List<Title>();
        public List<Artist> Composers {get; set; } = new List<Artist>();

    }

}

File: songUI/src/app/Interfaces/album.ts
import { Artist } from "./artist"
import { Songs } from "./songs"

export interface Album {
    id: number;
    albumName: string;
    composers: [];
    tracklist: []
}

/*"album": {
      "id": 3,
      "albumName": "Gorillaz",
      "trackList": [
        null
      ],
      "composers": []
    }
  } */
